http-tests.md

```diff
@@ -835,6 +836,27 @@ You may assert that the `name` property of the `user` object matches a given val
 
     $response->assertJsonPath('user.name', 'Steve Schoger');
 
+<a name="assert-json-missing-path"></a>
+#### assertJsonMissingPath
+
+Assert that the response does not contain the given path:
+
+    $response->assertJsonMissingPath($path);
+
+For example, if the JSON response returned by your application contains the following data:
+
+```js
```

Comment:
json ?
```suggestion
```json
```

Reply:
The file uses `js` everywhere, I think it's better to leave it as-is.